docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Introducing the TestCafe Jenkins plugin"
+permalink: /media/team-blog/:title.html
+isTeamBlog: true
+author: Pavel Redrukhin, Eugene Titerman
+---
+# Introducing the TestCafe Jenkins plugin
+
+TestCafe offers a number of debugging features, including the capability to automatically capture videos and screenshots during the testing process. You can record all your tests or only those that failed, generate multiple video files or stitch them together, and even single out specific interactions (see [the documentation](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html) for the full list of available options).
+
+But automating this process was not always easy. For a long time, Jenkins users had to manually match completed tests to their artifacts. The recent introduction of the [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) changes that. Links to screenshots and videos taken during the test now automatically appear on the Jenkins test results page. You don't even need to keep your testing server running — all the necessary files are stored right inside the Jenkins build folder.
```

Comment:
The first sentence is too "out of the blue", unexpected - we haven't said anything about automation yet... And now we go right in and say that it wasn't always "easy". Starting the entire thing with "but" is also not the best choice. 

```suggestion
You can easily review test screenshots and videos manually, but you'll mostly want recordings available somewhere within your automated testing process. Until recently, Jenkins users had to manually match completed tests to their artifacts. The newly implemented [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) changes that. Links to screenshots and videos taken during a test run now automatically appear on the Jenkins test results page. You don't even need to keep your test server running — all the necessary files are stored right inside the Jenkins build folder.
```

Reply:
Suggestion: 
``` markdown
Many TestCafe users take advantage of Jenkins — a CI/CD solution — to automate their tests. Until recently, they had to manually match the screenshots and videos taken by TestCafe to the individual test reports. The newly implemented [TestCafe Jenkins plugin](https://plugins.jenkins.io/testcafe/) simplifies this process. 
```

Rationale: Sub-optimal logic: we start the paragraph by mentioning the process of manually reviewing test artifacts, then talk about automation, then go back to the manual review process.

Reply:
That's good.